adserver/templates/adserver/advertiser/overview.html

```diff
@@ -43,7 +43,8 @@ <h3>{% trans 'There are a few steps to getting started with your first ad campai
               {% else %}
                 <span class="fa-li fa fa-square-o"></span>
               {% endif %}
-              <p>{% blocktrans %}Upload your ad creatives to your <a href="{{ flight_list_url }}">flights</a>.{% endblocktrans %}</p>
+              {% url 'support' as support_url %}
+              <p>{% blocktrans %}Upload your ad creatives to your <a href="{{ flight_list_url }}">flights</a>. Need help getting your ads setup? <a href="{{ support_url }}?subject=Help+with+ads">Let us know</a>. We're happy to help!{% endblocktrans %}</p>
```

Comment:
Should we link to the specs or the creatives PDF guide?

Reply:
Good idea.